### PR TITLE
Footer project text and logo

### LINKED ISSUE #26: Add logo and short about text to the footer
## User story
As a person discovering the site I need to know who is behind the site so that I can trust the information.

## Acceptance criteria
- [x] I can see a logo in the footer
- [x] I can see some information about the project in the footer

## Implementation notes & questions
Probably worth adding the responsive layout for footer "columns" with this if that hasn't been done in #27

### PR DESCRIPTION
**Pull request by ivan-kocienski-gfsc**
Fixes #26 

## Description

Footer has (rough) SVG logo and project text

@geeksforsocialchange/developers

### CODE REVIEW

**Comment by katjam** [comment]
@ivan-kocienski-gfsc When I said any logo would do, I meant a generic svg, not one from another client. :)

**Comment by katjam** [comment]
@undividual Do you have a TW logo to hand?

File: src/Theme/FooterTemplate.elm [hunk 1]
```diff
@@ -15,10 +15,14 @@ view model =
             translate model.language
     in
     footer [ css [ footerStyle ] ]
-        [ h2
+        [ img [ src "/images/dfl-logo.svg" ] []
+        , h2
             []
             [ text (t FooterText)
```
Comment (katjam) [inline src/Theme/FooterTemplate.elm:21]: I think we should remove this h2. It was a placeholder so we could "see the footer"

File: src/Theme/FooterTemplate.elm [hunk 2]
```diff
@@ -15,10 +15,14 @@ view model =
             translate model.language
     in
     footer [ css [ footerStyle ] ]
-        [ h2
+        [ img [ src "/images/dfl-logo.svg" ] []
```
Comment (katjam) [inline src/Theme/FooterTemplate.elm:18]: In the design, the text sits on top of the logo. Also see note about either using a dummy logo or one from their website or Emma.

**Review by katjam — CHANGES_REQUESTED** [review]

**Comment by undividual** [comment]
> 

To the best of my knowledge there is no TeamWilder logo. Would you like a Wildlife Trusts logo?
index

**Comment by katjam** [comment]
@undividual Yes, sorry that's what I meant. Do you have an svg?

**Comment by undividual** [comment]
> @undividual Yes, sorry that's what I meant. Do you have an svg?

No — you'll see I've emailed them to ask!

**Review by katjam — APPROVED** [review]
Looks great!

**Pull request merged**